test/rt-deadline_test

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+. common
+
+# Skipping this because of: https://github.com/canonical/rt-tests-snap/issues/8
+exit $SKIP
```

Comment:
Would be more consistent to do this also via a function, avoiding the need for the global var and name confusion.